omnibus/config/software/datadog-security-agent-policies.rb

```diff
@@ -38,4 +38,22 @@
 
   end
 
+  if linux?
```

Comment:
Nit: this `if` is not needed (at least for now), as this software definition is only added on linux: https://github.com/DataDog/datadog-agent/blob/b05aa49c616c8872766154e2de71b26608885d58/omnibus/config/projects/agent.rb#L205-L207

Reply:
Indeed. Fixed